sw/airborne/firmwares/rotorcraft/guidance/guidance_h.c

```diff
@@ -595,6 +595,8 @@ static void guidance_h_traj_run(bool in_flight)
 
 static void guidance_h_hover_enter(void)
 {
+  /* reset speed setting */
+  guidance_h_set_guided_vel(0., 0.);
```

Comment:
IMHO it would be better to directly set
```
guidance_h.sp.speed.x = 0;
guidance_h.sp.speed.y = 0;
```
here instead of resetting vel to zero only if you are in guided mode, set the xy speed bit and then clear it again in the next line...